src/rpc/RPCServer.ts

```diff
@@ -56,12 +58,21 @@ class RPCServer extends EventTarget {
    * path and `JSONRPCResponse` to `Uint8Array` on the reverse path.
    * @param obj.sensitive - If true, sanitises any rpc error messages of any
    * sensitive information.
+   * @param obj.streamKeepAliveTimeoutTime - Time before a connection is cleaned up due to no activity. This is the
+   * value used if the handler doesn't specify its own timeout time. This timeout is advisory and only results in a
+   * signal sent to the handler. Stream is forced to end after the timeoutForceCloseTime. Defaults to 60,000
+   * milliseconds.
+   * @param obj.timeoutForceCloseTime - Time before the stream is forced to end after the initial timeout time.
+   * The stream will be forced to close after this amount of time after the initial timeout. This is a grace period for
+   * the handler to handle timeout before it is forced to end. Defaults to 2,000 milliseconds.
```

Comment:
The client is tracking whether the server is idle. The server is tracking whether the handler is idle.

So these 2 properties in the `RPCServer` is really about the handler.

You want to call these:

1. `handlerTimeoutTime` - when this is exhausted we notify the handler by calling the abort controller
2. `handlerTimeoutGraceTime` - this is about if the handler is not aborting, then we force "abort" by closing the streams

The idea here is to make it clear that these times are about the handler.

In both cases, you could have `Infinity` as a value.

Reply:
This has been fixed.